ChangeLog.md

```diff
@@ -9,6 +9,10 @@
  * Make SSL "connection closed" accept failures a debug message.
  * Remove port from SSL connection failures so they log dedup.
  * Make ncct (telnet console) output thread safe (crash fix).
+ * Fix leak of thread name in SMR context.
+ * Add eventer_jobq_memory_safery_name().
```

Comment:
Also, for better Markdown formatting, please enclose function names in backticks. It help avoid editor syntax highlighting issues, since underscores are Markdown-special.